hips/utils/testing.py

```diff
@@ -58,5 +57,17 @@ def make_test_wcs_geometry(case=0):
             shape=(1000, 2000), coordsys='GAL',
             projection='AIT', cdelt=0.01, crpix=(1000, 500),
         )
+    elif case == 3:
```

Comment:
I suggest you move these cases simply to unit tests of `WCSGeometry` in `TestWCSGeometry`.
I.e. have test methods that instantiate a WCSGeometry by calling this method and then assert some properties of the resulting object.

The only reason for this `make_test_wcs_geometry` was to have something available for other tests.
We should keep the number of cases here as small as possible, i.e. simple unit tests for `WCSGeometry` go to `TestWCSGeometry`.